packages/dht/src/connection/websocket/ClientWebsocket.ts

```diff
@@ -23,7 +23,7 @@ export class ClientWebsocket extends EventEmitter<ConnectionEvents> implements I
         this.connectionId = new ConnectionID()
     }
 
-    public connect(address: string, selfSigned?: boolean): void {
+    public connect(address: string, selfSigned = false): void {
```

Comment:
Is it important to know for each connection request whether the certificate is `selfSigned`? If so, this should be required parameter. (And also the parameter name should tell that this is about a certificate.)

Reply:
At the moment there is only one case where selfSigned is set to true. It happens on the first ConnectivityRequest to an entry point when a node is started with auto certification enabled